CMakeLists.txt

```diff
@@ -844,6 +872,8 @@ if( PETSC_FOUND )
   
   
   ### add target for flow123d binary
+  #
+  # set(CMAKE_EXECUTABLE_FORMAT "ELF")
```

Comment:
IF it is at least hopefully working workaround for known bug in cmake, we should check cmake version and use command only for particular version and provide extensive comment including url of appropriate bug report.